components/push/android/src/main/java/mozilla/appservices/push/LibPushFFI.kt

```diff
@@ -42,43 +40,43 @@ internal interface LibPushFFI : Library {
     // free them).
 
     /** Create a new push connection */
-    fun push_connection_new(
-            server_host: String,
-            socket_protocol: String?,
-            bridge_type: String?,
-            registration_id: String,
-            sender_id: String?,
-            database_path: String,
-            out_err: RustError.ByReference
+    fun new_connection(
+        server_host: String,
+        socket_protocol: String?,
+        bridge_type: String?,
+        registration_id: String,
+        sender_id: String?,
+        database_path: String,
+        out_err: RustError.ByReference
     ): PushManagerHandle
 
-    /** Returns JSON string, which you need to free with push_destroy_string */
-    fun push_subscribe(
-            mgr: PushManagerHandle,
-            channel_id: String,
-            scope: String,
-            out_err: RustError.ByReference
+    /** Returns JSON string, which you need to free with destroyString */
+    fun subscribe(
```

Comment:
These functions are in a global namespace shared with all functions defined by all libraries used by the application, which is why we had them prefixed with `push_`. We should keep that, it serves a purpose even if it's a little ugly.